backend/app/schemas.py

```
@@ -163,6 +155,8 @@ class OrderOut(BaseModel):
     class Config:  
         from_attributes = True
 
+
+
 class OrderUpdate(BaseModel):
     status: st
```

Ruff (0.8.2)

Undefined name `st`

(F821)